packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -111,12 +113,15 @@ contract FastBridgeV2 is Admin, IFastBridgeV2, IFastBridgeV2Errors {
         bridgeTxDetails[transactionId].status = BridgeStatus.REFUNDED;
 
         // transfer origin collateral back to original sender
-        address to = transaction.originSender;
-        address token = transaction.originToken;
```

Comment:
Reading from memory just once should be more efficient, so let's get these two lines back?